### ISSUE TITLE
[ERROR] Package name unspecified

### ISSUE THREAD

**Issue by grayswandyr** [ocaml/opam-publish#61]
Hi
I'm trying to prepare a [package](https://github.com/grayswandyr/electrod) but I constantly get the `[ERROR] Package name unspecified` error. E.g.:
```
$ opam publish prepare electrod.0.1.1 https://github.com/grayswandyr/electrod/archive/0.1.1.tar.gz
[0.1.1.tar.gz] https://github.com/grayswandyr/electrod/archive/0.1.1.tar.gz downloaded
[ERROR] Package name unspecified
```
It's not very clear... what should I do?
Thanks!

**Comment by lpw25** [comment]
I also get this error.

**Comment by rjbou** [comment]
The name is taken from the opam file, not from the command-line (not like the version that is completed from command-line). A fix can be needed only in 0.3.5, it seems that in 2.0.0~beta name in command-line is handled. 
ping @AltGr

**Comment by rjbou** [comment]
handled in current version. thanks!
